src/elements/content-picker/stories/tests/ContentPicker-visual.stories.js

```diff
@@ -17,8 +25,204 @@ export const selectedEmptyState = {
         const selectedButton = canvas.getByRole('button', { name: '0 Selected' });
         await userEvent.click(selectedButton);
         await waitFor(() => {
-            expect(canvas.getByText('You haven’t selected any items yet.')).toBeInTheDocument();
+            expect(canvas.getByText("You haven't selected any items yet.")).toBeInTheDocument();
+        });
+    },
+    parameters: {
+        msw: {
+            handlers: [
+                http.get(`${DEFAULT_HOSTNAME_API}/2.0/folders/69083462919`, () => {
+                    return HttpResponse.json(mockRootFolder);
+                }),
+            ],
+        },
+    },
+};
+
+export const emptyFolder = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        await waitFor(
+            () => {
+                // Verify empty folder state
+                expect(canvas.getByText('This folder is empty')).toBeInTheDocument();
+            },
+            {
+                timeout: SLEEP_TIMEOUT,
+            },
+        );
+    },
+    parameters: {
+        msw: {
+            handlers: [
+                http.get(`${DEFAULT_HOSTNAME_API}/2.0/folders/69083462919`, () => {
+                    return HttpResponse.json(mockRootFolder);
+                }),
+            ],
+        },
+    },
+};
+
+export const emptySelectionMode = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        // Click to enter selection mode
+        const chooseButton = canvas.getByLabelText('Choose');
+        await userEvent.click(chooseButton);
+
+        await waitFor(
+            () => {
+                // Verify empty folder state in selection mode
+                expect(canvas.getByText('This folder is empty')).toBeInTheDocument();
+                expect(chooseButton).toBeDisabled();
+            },
+            {
+                timeout: SLEEP_TIMEOUT,
+            },
+        );
```

Comment:
you dont need to waitfor, you just need to check if the chooseButton is disabled. dont need to check for text because the ui should not change when clicking chooseButton since the button is disabled